app/renderer/js/utils/system-util.ts

```diff
@@ -32,14 +43,9 @@ export function getOS(): string {
 		return '';
 	}
 }
-
-export function setUserAgent(webViewUserAgent: string): void {
```

Comment:
We need to use this with the session's UA. We are using `ZulipElectron` to detect the desktop app. Check out -
https://github.com/zulip/zulip/search?q=ZulipElectron&unscoped_q=ZulipElectron

cc @timabbott

Reply:
@akashnimare Maybe we should refine `test_insecure_desktop_app(self)` itself? As looking at L101, isn't this exactly what @andersk pointed out? As this forces us to append `ZulipElectron/${app.getVersion()}` to the UA.

https://github.com/zulip/zulip/blob/17723b16177663a558c03ef6a41a926163ce4241/zerver/tests/test_compatibility.py#L101